geode-core/src/main/java/org/apache/geode/management/internal/cli/commands/LauncherLifecycleCommands.java

```diff
@@ -771,11 +771,18 @@ public Result statusLocator(
               CliStrings.STATUS_SERVICE__GFSH_NOT_CONNECTED_ERROR_MESSAGE, LOCATOR_TERM_NAME));
         }
       } else {
-        final LocatorLauncher locatorLauncher =
-            new LocatorLauncher.Builder().setCommand(LocatorLauncher.Command.STATUS)
-                .setBindAddress(locatorHost).setDebug(isDebugging()).setPid(pid)
-                .setPort(locatorPort).setWorkingDirectory(workingDirectory).build();
+        final LocatorLauncher locatorLauncher;
 
+        if ((locatorHost == null) && (locatorPort == null) && (workingDirectory == null)) {
```

Comment:
I think that if a value is given for locatorPort but not for locatorHost, `locatorLauncher` will just have a null value for locatorPort, with no errors (in the `else` statement). Do you mean there will be an error in a different part of the code?